File: Python-Assignment/MortgageCalculator/mortgageCalculator.py
principal = float(input("Enter the principal amount: "))

rate = int(input("Enter rate: "))

no_of_years = int(input("Enter number of years: "))

annual_rate = rate / 100 / 12

duration = no_of_years * 12

numerator = annual_rate * (1 + annual_rate) ** duration

denominator = ((1 + annual_rate) ** duration) - 1

monthly_payment = principal * (numerator / denominator)

print("Monthly payment: ", monthly_payment)

File: Python-Assignment/MortgageCalculator/MortgageCalculator.java
import java.util.Scanner;

public class MortgageCalculator{
   public static void main(String []args){

	Scanner input = new Scanner (System.in);

	System.out.print("Enter Principal Amount: ");
	double principal = input.nextDouble();

	System.out.print("Enter number of years: ");
	double noOfYears = input.nextDouble();

	System.out.print("Enter Annual rate: ");
	double annualRate = input.nextDouble();

	double duration = noOfYears * 12;

	double monthlyRate = annualRate / 12;

	double percentageMonthlyRate = monthlyRate / 100;

	double percentageMonthlyRateRaisedToPowerDuration = Math.pow((1+percentageMonthlyRate), duration);

	double numerator = percentageMonthlyRate * percentageMonthlyRateRaisedToPowerDuration;

	double denominator = percentageMonthlyRateRaisedToPowerDuration - 1;

	double monthlyPayment = principal * (numerator / denominator);

	System.out.printf("The Monthly payment is: %.2f", monthlyPayment);
   }
}